Allwmake

```diff
@@ -3,7 +3,9 @@ set -e -u
 
 ########## CONFIGURATION - You may want to change these ########################
 
-# Optional: Preprocessor flags ("-DADAPTER_DEBUG_MODE" enables debug messages)
+# Optional: Preprocessor flags
+# "-DADAPTER_DEBUG_MODE" enables debug messages
+# "-DADAPTER_TIMING_MODE" enables time measurements
```

Comment:
From the user perspective, this sounds a bit like the adapter is doing something differently ('mode'), but it is actually doing the same, just measuring. Maybe better use
```suggestion
# "-DADAPTER_ENABLE_TIMINGS" enables time measurements
```

Reply:
Good point. Renamed everywhere.